ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### Description
+
+[Description of the issue]
+
+### Steps to Reproduce
+
+1. [First Step]
+2. [Second Step]
+3. [etc...]
+
+**Expected behavior:** [What you expect to happen]
+
+**Actual behavior:** [What actually happens]
+
+**Reproduces how often:** [What percentage of the time does it reproduce?]
+
+### Versions
+
+Please report the outputs of `psi4 --version`, `python --version`, `numpy --version`, and `scipy --version`.
+
+### Additional Information
+
+Any additional information, configuration or data that might be necessary to reproduce the issue.
```

Comment:
Since we want to have folks using our environment for their own development & prototyping, I thought that folks would feel more comfortable if there is a standard way in which to report issues.  Some issues may end up being farmed elsewhere, but I still think it's safer to have than not.  Could simplify it to a description of the error and version info, however, as those are probably the most crucial pieces of info.